python/cudf/dataframe/numerical.py

```diff
@@ -104,6 +104,12 @@ def unary_operator(self, unaryop):
 
     def unordered_compare(self, cmpop, rhs):
         lhs, rhs = numeric_normalize_types(self, rhs)
+        if cmpop == 'eq':
+            if len(self) != len(rhs):
+                return False
+        elif cmpop == 'ne':
+            if len(self) != len(rhs):
+                return True
```

Comment:
Why does this go into unordered compare, but not into ordered compare? It seems like it should go into `numeric_column_compare`?

Reply:
This is actually wrong period, this should be for `.equals` and not an elementwise operation. Thanks!